### PR TITLE
Building and releasing with fastlane

### PR DESCRIPTION
**Pull request by stel**
Use fastlane for building, testing and submitting (ITC, Hockeyapp) builds on CI.

Currently needs some additional Jenkins setup:
- Installing Mac *** certificates
- Setup ITC credentials

fixes https://github.com/realm/realm-browser-osx/issues/17

### CODE REVIEW

**Comment by jpsim** [comment]
Why is this marked as "in progress" @stel? What's left to do to get this in a reviewable state?

@larkost could you please review this and work with @stel to perform whatever infrastructure changes are necessary to support doing this from our CI workers? Thanks!

**Comment by stel** [comment]
@jpsim I just want to make sure that everything works properly on CI first, currently certificates and ITC credentials are left. I'll create separate issues for this.

Anyway any feedback is appreciated.

**Comment by jpsim** [comment]
Ok, generally when PRs are marked as "In Progress", it's very useful to share why they're marked as such.

**Comment by stel** [comment]
Everything works fine on CI, but submitting builds will be implemented in a separate Jenkins job because of security reasons.

It's also possible to build and submit builds from the local machine by defining the necessary environment variable and executing `fastlane build submit_to_itc:true symbol_to_hockey:true` (you will also need all certificates installed locally).

Any feedback is appreciated.

/cc @larkost @radu-tutueanu

File: fastlane/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+fastlane documentation
```
Comment (radu-tutueanu) [inline fastlane/README.md]: I see that this is autogenerated. Documenting the ENV variables in a readme would be helpful.

**Pull request merged**